profiler/src/profile_gemm_multiply_multiply.cpp

```diff
@@ -76,11 +77,13 @@ int profile_gemm_multiply_multiply(int argc, char* argv[])
     int n_warmup      = 1;
     int n_iter        = 10;
     uint64_t rotating = 0;
-    if(argc == 19)
+    int KBatch        = 1;
```

Comment:
We can iterate KBatchList when KBatch input is <0

Reply:
Done